android/templates/release-0.5/src/main/java/com/badoo/ribs/template/rib_with_view/foo_bar/FooBarView.kt

```diff
@@ -37,11 +39,20 @@ class FooBarViewImpl private constructor(
     ) : FooBarView.Factory {
         override fun invoke(deps: Nothing?): (ViewGroup) -> FooBarView = {
             FooBarViewImpl(
-                com.badoo.ribs.customisation.inflate(it, layoutRes)
+                inflate(it, layoutRes)
             )
         }
     }
 
-    override fun accept(vm: FooBarView.ViewModel) {
+    // feel free to check https://badoo.github.io/MVICore/extras/modelwatcher/ for usage examples
```

Comment:
Should we force usage of ModelWatcher for customers of the Framework?
Maybe we need to mention it once with context of applying changes for separate fields of the model.

But I am not sure, that we need to force using it. You can have simple RIB with one textField and probably you don't want to use modelWatcher for this case. We are adding more and more complex stuff which can be repulsive for customers of the BadooRIBs.

Reply:
You are probably right, thanks. Can we make it not feel forced, just optional and encouraged?